src/hotspot/share/gc/shenandoah/shenandoahHeapRegionClosures.cpp

```diff
@@ -1,5 +1,4 @@
 /*
- * Copyright (c) 2021, Red Hat, Inc. All rights reserved.
```

Comment:
This had me confused, but I verified that the deletion of the Copyright line is indeed correct based on what I can tell from `git annotate`.

Also the way the rename shows up in the diffs here is pretty subtle.

Reply:
Yes, git has a mind of its own when it comes to renames.